markhor_flippers/src/markhor_hw_interface_flippers.cpp

```diff
@@ -4,6 +4,31 @@
 
 #include "markhor_hw_interface_flippers.hpp"
 
+static float fr_target = 0;
```

Comment:
Why not use member variable instead of global static variable ?

Reply:
No, the idea is to use **member** variable instead of global static variable. In other words, we will put the variable inside the class instead of using global variables. 

Also, I was more asking why use `global static`, maybe you have a good reason, that I'm not seeing.

Reply:
Donc mettre les var dans le `main` ? J'avais aucune bonne raison

Reply:
No, I am talking about member variable like inside a class refer to this : https://en.cppreference.com/w/cpp/language/data_members